/flo_base_diff:
  ros__parameters:
    debug: false
    pub_joint_states: true
    hb_timeout: 1.0
    hb_hz: 10.0
    left_wheel_joint_index: 0
    right_wheel_joint_index: 1
    wheel_diameter: 0.26  # meters
    wheel_to_wheel_separation: 0.62  # meters
    wheel_separation_multiplier: 1.0
    cmd_vel_timeout_enabled: true
    cmd_vel_timeout: 0.5
    publish_odom: true
    publish_pose_in_odom: true
    odom_hz: 30.0
    drive_joint_timeout: 1.0
    enable_observer: true
    observer_hz: 1.0
    enable: false

/serial_port_drive:
  ros__parameters:
    device: "/dev/ttyUSB0"
    device_name: "serial_port_drive"
    baud_rate: 460800
    # carriage_return: false
    # raw_in: false

/keyboard_controller:
  ros__parameters:
    linear_speed: 0.36
    angular_speed: 0.5
    update_rate: 0.1